nativelib/src/main/scala/scala/scalanative/native/Platform.scala

```diff
@@ -0,0 +1,14 @@
+package scala.scalanative
+package native
+
+@extern
+object Platform {
+  @name("scalanative_platform_is_windows")
+  def isWindows(): Boolean = extern
+
+  @name("scalanative_windows_get_user_lang")
+  def windowsGetUserLang(): CString = extern
+
+  @name("scalanative_windows_get_user_country")
+  def windowsGetUserCountry(): CString = extern
+}
```

Comment:
This is not yet polished enough to be in public API namespace yet. Lets include it in `scala.scalanative.runtime` for now.